s4e-web/cypress/integration/auth/auth-login.spec.ts

```diff
@@ -1,16 +1,17 @@
 /// <reference types="Cypress" />
 
-import { Login } from '../../page-objects/login/login.po';
+import { Login } from '../../page-objects/auth/login.po';
 
 context('Auth login', () => {
   beforeEach(() => {
     cy.fixture('users/zkMember.json').as('zkMember');
     cy.fixture('users/zkAdmin.json').as('zkAdmin');
     cy.fixture('users/admin.json').as('superAdmin');
+    cy.fixture('users/userToRegister.json').as('userToRegister')
   });
 
   beforeEach(() => {
```

Comment:
`beforeEach` should be replaced with `before`

Reply:
with `before` only first test is passed, others are failed.